src/Chapter10.Tests/Listing10.23.RegisteringAFinalizerWithProcessExit.Tests.cs

```diff
@@ -2,62 +2,30 @@
 using System;
 using System.Diagnostics;
 using System.IO;
-using Chapter10.Tests.PowerShellTestsUtilities;
+using AddisonWesley.Michaelis.EssentialCSharp.Shared.Tests;
+
 using System.Text.RegularExpressions;
 
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter10.Listing10_23.Tests
 {
     [TestClass]
-
     public class DisposeTests
     {
-
         static string Ps1Path { get; } = Path.GetFullPath("../../../../Chapter10/Listing10.23.RegisteringAFinalizerWithProcessExit.ps1", Environment.CurrentDirectory);
```

Comment:
```suggestion
        static string Ps1Path { get; } = Path.GetFullPath(Path.Combine("..,..,..,..,Chapter10,Listing10.23.RegisteringAFinalizerWithProcessExit.ps1"), Environment.CurrentDirectory);
```
Shouldn't this use commas to make it cross-plat compatible?